x-pack/plugins/spaces/server/routes/api/external/copy_to_space.test.ts

```diff
@@ -365,58 +346,6 @@ describe('copy to space', () => {
       );
     });
 
-    it('does not allow namespace agnostic types to be copied (via "supportedTypes" property)', async () => {
```

Comment:
Do we have a test for this elsewhere, or should this be re-introduced here?

Reply:
This was changed in 3538c2b. There isn't a separate test case for it, but it is exercised in every test. I can change that if you think it would be better to have an explicit test case.

Reply:
Thanks, that's good enough for me!